test/pubsub_test.cpp

```diff
@@ -36,16 +36,16 @@ SOFTWARE.
 
 const char topic[] = "benchmark_topic";
 const int SECONDS = 10;
-const int VECTOR_SIZE = 10 * 1024 * 1024;
+const int VECTOR_SIZE = 1024 * 1024;
 
 shm::stats::Welford per_second_lag;
 
 class BenchmarkMsg : public shm::BaseMsg {
  public:
-  int number{};
+  uint8_t number{};
   std::vector<uint8_t> arr;
   SHM_PACK(number, arr);
-  explicit BenchmarkMsg(int n) : number(n) {
+  explicit BenchmarkMsg(uint8_t n) : number(n) {
```

Comment:
This will cause a truncation error when passing in `VECTOR_SIZE` of 2^20. Revert.